bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/type/MetadataUtils.java

```diff
@@ -77,9 +77,16 @@ private static void loadStandardDatapoints() {
                 BufferedReader reader = new BufferedReader(new InputStreamReader(stream))) {
             String line;
             while ((line = reader.readLine()) != null) {
-                if (StringUtils.trimToNull(line) != null && !StringUtils.startsWith(line, "#")) {
-                    String channelType = StringUtils.trimToNull(StringUtils.substringBefore(line, "|"));
-                    String datapointName = StringUtils.trimToNull(StringUtils.substringAfter(line, "|"));
+                if (!line.trim().isEmpty() && !line.startsWith("#")) {
+                    String[] parts = line.split("\\|");
+                    String channelType = null;
+                    String datapointName = null;
+                    if (parts.length > 0) {
```

Comment:
Not sure if this is an issue. But if the input is something like `"|datapointName"` then only 1 result will be returned and `datapointName` is stored in `channelType`.

Reply:
The lines are read from a properties file that is manually created and included in the binding's jar file. In the file, no entry starts with "|" and it would cause problems if the channelType would be empty.